cs_api_io/cs_api_io.ml

```diff
@@ -77,3 +77,25 @@ let get_response {Response.code; body} =
   else
     let message = Printf.sprintf "HTTP response: %d\n%s" code body in
     Lwt_result.fail message
+
+let get_error_message (errors : Yojson.Safe.t list) =
+  match CCList.hd errors with
```

Comment:
This will raise an exception if the list of errors is empty. Could we catch that?